src/manager.rs

```diff
@@ -228,37 +241,43 @@ impl Manager {
 
     pub fn enable_poll_with_intensity(&mut self, value: &ValueID, intensity: u8) -> bool {
         unsafe {
-            extern_manager::manager_enable_poll_with_intensity(self.ptr, value.ptr, intensity)
+            let cvid = extern_value_id::value_id_from_packed_id(value.home_id, value.id);
```

Comment:
So you're talking about the "value_id_from_packed_id" name?
I think it needs to be value_id_xxx since its the name of a C function. Did you just want to call it "value_id"?